### PR TITLE
Fixed installation page 

### PR DESCRIPTION
**Pull request by skjnldsv**
- Removed jquery scss
- Switched to setup.css
- Disable scss files compiling when displaying the update page
- Improved setup css
- Fixed loading failure of other styles on setup & update page
- Improved scss compiler with an ignore scss compilation option

Following: #2992, #2984, #2050 
Refs: #1805

### CODE REVIEW

File: lib/private/TemplateLayout.php [hunk 1]
```diff
@@ -190,16 +187,20 @@ public function __construct( $renderAs, $appId = '' ) {
 	 * @param array $styles
 	 * @return array
 	 */
-	static public function findStylesheetFiles($styles) {
+	static public function findStylesheetFiles($styles, $compileScss=true) {
```
Comment (MorrisJobke) [inline lib/private/TemplateLayout.php]: Please add spaces around the `=`

**Review by MorrisJobke — DISMISSED** [review]

**Review by MorrisJobke — DISMISSED** [review]
Login page now looks like this:

bildschirmfoto 2017-01-10 um 11 20 27

**Comment by skjnldsv** [comment]
@MorrisJobke You found a discutable point.
Maybe we should create a single log-in-update-setup.scss file for the three of them ?

I feel like we could really improve the scss of those three file (or at least setup & update) because there is too much in common.
I will see what i can do. The scss merge was supposed to be a cleanup not something we can't handle anymore :)

**Comment by skjnldsv** [comment]
Okay, I need review now.
i think I've got everything covered.

@MorrisJobke @rullzer @ChristophWurst @LukasReschke

**Comment by skjnldsv** [comment]
@ChristophWurst I couldn't test the 2fa pages since I can't enable the app on my nc instance.

File: lib/private/TemplateLayout.php [hunk 2]
```diff
@@ -190,16 +188,20 @@ public function __construct( $renderAs, $appId = '' ) {
 	 * @param array $styles
 	 * @return array
 	 */
-	static public function findStylesheetFiles($styles) {
+	static public function findStylesheetFiles($styles, $compileScss = true) {
```
Comment (ChristophWurst) [inline lib/private/TemplateLayout.php]: "public static …" sounds better 😀

**Review by ChristophWurst — COMMENTED** [review]
installation page works on FF and Chrome
2FA page still looks the same

File: lib/private/TemplateLayout.php (hunk 2 above)
Comment (skjnldsv) [inline lib/private/TemplateLayout.php]: 😆

**Comment by skjnldsv** [comment]
I restarted the test.
Can we review this please?

@MorrisJobke @ChristophWurst @LukasReschke @jancborchardt

**Review by MorrisJobke — DISMISSED** [review]
Password strength meter is not aligned anymore: 
bildschirmfoto 2017-01-20 um 23 01 14

**Comment by MorrisJobke** [comment]
* [ ] download button on public link page looks out of place
bildschirmfoto 2017-01-21 um 16 59 17
* [ ] login page looks a broken:
bildschirmfoto 2017-01-21 um 17 00 56
* [ ] password protected link page:
bildschirmfoto 2017-01-21 um 17 02 01
* [ ] LDAP wizard: buttons look a bit too big compared to input (maybe okay)
* [ ] LDAP wizard: copy button icon looks a bit too big:
bildschirmfoto 2017-01-21 um 17 04 02


Sorry @skjnldsv for not finding those things earlier.

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Review by LukasReschke — APPROVED** [review]

**Comment by skjnldsv** [comment]
Thanks @LukasReschke 💘

**Comment by LukasReschke** [comment]
😘 😉

**Pull request merged**